### PR TITLE
Add pandoc package

### PR DESCRIPTION
**Pull request by JanSchulz**
This package is taken from https://github.com/conda/conda-recipes
and only updated to the latest version.

This is part of my plan to get pdf building working in the notebook on windows... A miktex package is next...

The original conda packaging is by asmeurer,  @jschueller and @ccordoba12 (Only linked the people which updated the package recently: https://github.com/conda/conda-recipes/commits/master/pandoc).

### CODE REVIEW

**Comment by JanSchulz** [comment]
WTF?

```
Error: MD5 sums mismatch for download: https://conda.anaconda.org/conda-forge/linux-64/smmap-0.9.0-py34_0.tar.bz2 (d41d8cd98f00b204e9800998ecf8427e != 5ece8f25e951bb44a2a5e302345d4adf)
[...]
Error: MD5 sums mismatch for download: https://repo.continuum.io/pkgs/free/linux-64/conda-build-1.18.2-py34_0.tar.bz2 (d41d8cd98f00b204e9800998ecf8427e != f1be65bb62a6416f70ad3960e377bac5)

**Comment by jjhelmus** [comment]
The md5 issue is caused by conda 4.0.3, conda/conda#2241.  Should be fixed now that conda 4.0.4 is released.

**Comment by jjhelmus** [comment]
Thanks for this @JanSchulz, pandoc has always been a pain to install for me.  I'm guessing that building from source using stack is quite involved hence the choice to repackage the binaries?

File: recipes/pandoc/notes.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* The Linux package doesn't work on systems older than CentOS 6
+* The OS X package works on 10.7+
+* The Windows package works on 32 and 64 bit archs
```
Comment (jjhelmus) [inline recipes/pandoc/notes.md]: Does this information get included on the Anaconda.org package description,  If not could it be added to the summary section of the meta.yaml file?

Comment (JanSchulz) [inline recipes/pandoc/notes.md]: It's in the recipe directory, but if that should be in any way user visible, it should be in the meta.yml...

But: IMO it's pretty much not interesting for users? 10.7 was released in 2011, windows is simple just states "it works everywhere" and Centos6 was also released in 2011...

**Comment by JanSchulz** [comment]
@jjhelmus an [older version of the recipe](https://github.com/conda/conda-recipes/commits/master/pandoc) installed the linux one form source, that was quite involved as it needs a haskel environment.

An idea to make it clear that this is a repackaged pandoc, we could name it "pandoc-repackaged", but until conda adds better dependency specs (e.g. in this case alternatives like `pandoc | pandoc-repackaged`, it's IMO not worth the pain...

I will add some info that it is a repackaged version to the recipe file...

**Comment by JanSchulz** [comment]
All builds pass, IMO this is ready (appveyor seems to be dead? But an older versions built fine)...

**Comment by ocefpaf** [comment]
> appveyor seems to be dead?

I guess you PR is next in the queue. (My fault.  I am sending many PRs today.)

**Comment by ccordoba12** [comment]
Just a quick note to say that we (at Continuum) found much easier to just repackage Pandoc installers as conda packages, instead of trying to compile it from source (which is really hard).

**Comment by ccordoba12** [comment]
Another thing is that there's no Linux 32 support because there are not Pandoc packages for that platform.

**Comment by ocefpaf** [comment]
> Just a quick note to say that we (at Continuum) found much easier to just repackage Pandoc installers as conda packages, instead of trying to compile it from source (which is really hard).

I understand that.  It "feels wrong" though, but I am not against it.  I look forward to see pandoc in the channel. (@JanSchulz are you done here? Ready to merge?)

**Comment by JanSchulz** [comment]
@ocefpaf imo yes...

**Pull request merged**